lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -18,15 +18,35 @@ class AmazonMarketplaceWebService < Service
     XML_FAILURE_RESPONSE = { :success => FAILURE }.freeze
 
     ENDPOINTS = {
-      :ca => 'mws.amazonservices.ca',
+      :au => 'mws.amazonservices.com.au',
+      :br => 'mws.amazonservices.com',
+      :ca => 'mws.amazonservices.com',
       :cn => 'mws.amazonservices.com.cn',
-      :de => 'mws-eu.amazonservices.ca',
-      :es => 'mws-eu.amazonservices.ca',
-      :fr => 'mws-eu.amazonservices.ca',
-      :it => 'mws-eu.amazonservices.ca',
+      :de => 'mws-eu.amazonservices.com',
+      :es => 'mws-eu.amazonservices.com',
+      :fr => 'mws-eu.amazonservices.com',
+      :in => 'mws.amazonservices.in',
+      :it => 'mws-eu.amazonservices.com',
       :jp => 'mws.amazonservices.jp',
-      :uk => 'mws-eu.amazonservices.ca',
-      :us => 'mws.amazonservices.com'
+      :mx => 'mws.amazonservices.com',
+      :uk => 'mws-eu.amazonservices.com',
+      :us => 'mws.amazonservices.com',
+    }.freeze
+
+    MARKETPLACE_IDs = {
```

Comment:
I know you mentioned ignoring the 👮 in the name of consistency, but I don't see another instance of `_IDs`. Can you make the `s` uppercase here?

Reply:
ah, missed this one. Thanks